templates/account.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% block body %}
+<h1>Create an Account:</h1>
+<form action="/user/add" method="POST">
+    <font color='red' id='msg'></font><br>
+    <label>Username: </label><input type="text" name="handle" class="form-control"/><br/>
+    <label>Password: </label><input type="password" name="password" class="form-control"/><br/>
+    <input type="submit" class="btn btn-block btn-warning" value="Submit"><br/>
+</form>
+<script>
+var state = '{{ request.args.get("success") }}';
+if(state == 'create0'){
+    id('msg').innerHTML = 'You cannot create an account with this username.';
+}
```

Comment:
I'm thinking about this again, and I think you should use Flask's `flash`: http://flask.pocoo.org/docs/0.11/patterns/flashing/

Main advantage is that you take advantage of an existing, tested solution for the problem of errors, instead of creating your own.